python/src/opendp/prelude.py

```diff
@@ -47,5 +47,6 @@
 from opendp.accuracy import *
 from opendp.core import new_function, new_queryable
 from opendp.context import *
+import opendp.polars
```

Comment:
I don't think any of the python code uses this, right now, so could leave it out of this PR? At one point, I think we were thinking of including this method and others under a new `dp.x` namespace, which would have all the methods which require optional installs?

Reply:
On second thought, I think this does make at least some sense. I added this because otherwise the plugin namespace won't get registered. You still can't access any imports from that module in the prelude.